src/tests/verify_resources.py

```diff
@@ -436,6 +439,26 @@ def test_14_res_contacts_match(rgs, rgfns, contacts):
 
     return errors
 
+def test_15_FACILITY_SITE_check():
```

Comment:
If you notice the rest of the `test_*` function names, none of them ends with "check" or similar - they are all tests.  You can drop the `_check` suffix.  If you want to be more specific you can call it `test_15_facility_site_files`, or make two separate tests (one for facility and one for site), or even just work these tests for `SITE.yaml` and `FACILITY.yaml` into `test_6_site`.